### PR TITLE
Integrate libQasm 0.6.0 into OpenSquirrel

### PR DESCRIPTION
**Pull request by elenbaasc**
…the side of libQasm!]

### CODE REVIEW

**Comment by elenbaasc** [comment]
In `cqasm.v3x.primitives` the primitives are still `Real` iso `Float`. This requires changes on the side of LibQASM.

File: test/test_integration.py [hunk 1]
```diff
@@ -88,28 +88,27 @@ def test_qi(self):
             version 3.0
 
             // This is a single line comment which ends on the newline.
-            // The cQASM string must begin with the version instruction even before any comments.
+            // The cQASM string must begin with the version instruction (apart from any preceding comments.
```
Comment (juanboschero) [inline test/test_integration.py:91]:
```suggestion
            // The cQASM string must begin with the version instruction (apart from any preceding comments).
```

File: test/test_integration.py [hunk 2]
```diff
@@ -119,49 +118,51 @@ def test_qi(self):
         myCircuit.decompose(decomposer=McKayDecomposer)
         output = str(myCircuit)
 
+        print(output)
```
Comment (juanboschero) [inline test/test_integration.py:121]:
```suggestion
        
```

**Comment by elenbaasc** [comment]
The `pyproject.toml` file has been updated such that OpenSquirrel now uses LibQASM v0.6.1.

**Review by juanboschero — APPROVED** [review]

**Pull request merged**